src/plugins/irc-events/connection.js

```diff
@@ -0,0 +1,75 @@
+var _ = require("lodash");
+var identd = require("../../identd");
+var Msg = require("../../models/msg");
+
+module.exports = function(irc, network) {
+	var client = this;
+
+	client.emit("msg", {
+		chan: network.channels[0].id,
+		msg: new Msg({
+			text: "Network created, connecting to " + network.host + ":" + network.port + "..."
+		})
+	});
+
+	irc.on("raw socket connected", function() {
+		identd.hook(irc.connection.socket, network.username);
```

Comment:
> Complaining about the most important bit of the code!

I'm not complaining, I'm reviewing :-)

> Looks like this was done to support node 0.12, same thing here

Dang. Moving on then!
Maybe we should add the same comment in the `hook` function if we ever have to change that file again.

**This comment comment can now be ignored.**